connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -1378,4 +1382,16 @@ public void refreshEmotes(List<UUID> emotes) {
             player.sendUpstreamPacket(emoteList);
         }
     }
+
+    public void registerDownstreamPluginChannels(Collection<String> channels){
```

Comment:
I think this is overkill. As of right now, the only channel we care about is the emote channel. Just have a boolean tracking that.

Reply:
Also
```suggestion
    public void registerDownstreamPluginChannels(Collection<String> channels) {
```

Reply:
Yes, we track everything.
But will we ever care about other plugin channels?
And if geyser extensions branch will be merged, maybe someone will need plugin channels.

Reply:
I like solutions, what we won't need to change in the close future.

Reply:
If we wanted to add more plugin channels, we can add a boolean for each one in the future, and potential extension authors will likely be able to track their own if they want. Us using booleans saves multiple strings, the majority of which we don't need to worry about.